pkgs/development/tools/parsing/tree-sitter/default.nix

```
@@ -193,7 +197,7 @@ rustPlatform.buildRustPackage {
   preBuild = lib.optionalString webUISupport ''
     mkdir -p .emscriptencache
     export EM_CACHE=$(pwd)/.emscriptencache
-    bash ./script/build-wasm --debug
+    cargo run --package xtask -- build-wasm
```

I failed to mention it in the earlier issue but it looks like the new xtask script of build-wasm still supports the `--debug` flag (the comment above this `preBuild` mentions that it is useful).

https://github.com/tree-sitter/tree-sitter/blob/17471bdfcca5da6c68d519a4cc070d3db4d2c580/xtask/src/main.rs/#L74-L77

```suggestion
    cargo run --package xtask -- build-wasm --debug
```

---

Whoops, thank you. Maybe I should've let you submit the PR ...

---

Nono I'm glad you did!